src/cli.yaml

```diff
@@ -0,0 +1,395 @@
+            - Julian Ganz <neither@nut.email>
+        args:
+            - parent:
+                help: Parent
+                index: 1
+                multiple: false
+            - tree-init-hash:
+                help: Tree init hash
+                index: 2
+                multiple: false
+            - abbrev:
+                short: a
+                long: abbrev
+                help: Abbreviate issue hash
+                multiple: false
+                takes_value: false
+            - initial:
+                short: I
+                long: initial
+                help: Only list initial message
+                multiple: false
+                takes_value: false
+            - tree:
+                short: g
+                long: tree
+                help: List messages as a tree
+                multiple: false
+                takes_value: false
+            - msgtree:
+                short: t
+                long: message-tree
+                help: Show message tree (only subjects)
+                multiple: false
+                takes_value: false
+                conflicts_with:
+                    - verify
+                    - decorate
+                    - tree
+                    - initial
+            - verify-gpg:
+                short: V
+                long: verify-gpg
+                help: Verify gpg signatures
+                multiple: false
+                takes_value: false
+            - decorate:
+                short: d
+                long: decorate
+                help: Decorate (show references)
+                multiple: false
+                takes_value: false
+            - format:
+                short: F
+                long: format
+                help: Alternative format (see git-log(1))
+                multiple: false
+                takes_value: true
+                value_names:
+                    - format
+
+    - tag:
+        about: >
+                 A tag is a key-value pair of form: '<key>=<value>', where the
+                 <value> part can be in quotes to permit spaces.
+                 Multiple -s are permitted, later specified values will override
+                 former. Alter tags of an issue.
+        version: 0.1.0
+        authors:
+            - Matthias Beyer <mail@beyermatthias.de>
+            - Julian Ganz <neither@nut.email>
+        args:
+            - issue-hash:
+                help: Issue hash
+                index: 1
+                multiple: false
+            - list:
+                short: l
+                long: list
+                help: List Tags. Lists all tags which were introduced in the discussion thread of the issue
+                multiple: false
+                takes_value: false
+                conflicts_with:
+                    - set-status
+            - set-status:
+                short: s
+                long: status
+                help: Set a status tag. Key-value pair expected.
```

Comment:
If we call it "status" it should just set `Git-status`. For other metadata, the interface of `reply` should be mirrored.